src/handlers/gitHook.ts

```diff
@@ -0,0 +1,80 @@
+import { APIGatewayEvent } from "aws-lambda";
+import { BitbucketWebhookEvent, DataItem, GitTag, State } from "stackjanitor";
+import { DataModel, dataModel } from "../data/DynamoDataModel";
+import { logger } from "../logger";
+import { handleDataItem } from "./monitorCloudFormationStack";
+import { findStacksFromTag } from "../helpers";
+
+export const SEARCH_KEY = "tags";
+
+export enum Response {
+  IGNORE = "Ignored as PR is not in Merged or Declined state",
```

Comment:
Can we **Title** Case these like we do in the rest of the project?

Reply:
done